composer.json

```diff
@@ -18,15 +18,20 @@
             "homepage":  "http://everzet.com"
         }
     ],
-
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/oleg-andreyev/driver-testsuite.git",
+            "no-api": true
+        }
+    ],
     "require": {
-        "php":                       ">=5.3.1",
+        "php":                       ">=5.4",
         "behat/mink":                "~1.7@dev",
         "instaclick/php-webdriver":  "~1.1"
     },
-
     "require-dev": {
-        "mink/driver-testsuite": "dev-master"
+        "mink/driver-testsuite": "dev-drop-php-53-force-phpunit-less-8"
```

Comment:
Looks like a debug code.

If you need another PR in test suite to be merged before this one, then please indicate it here.

Reply:
see https://github.com/minkphp/driver-testsuite/pull/37

Reply:
Associated PR merged. Please revert this line and any connected lines, that override used package repositories.

Reply:
Changes were pushed 1a00312